blog/2025-07-01-khepri-default/index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Khepri is now the default metadata store"
+tags: ["khepri"]
+authors: [mgary]
+---
+
+## Khepri is now the default metadata store
+
+Khepri, the new raft-based RabbitMQ metadata store, became fully supported with RabbitMQ 4.0. Since then, it has been extensively tested and shown to be an improvement over the old mnesia-based metadata store in juts about every regard, and we have performed [a number of benchmarks](#khepri-performance-improvements), showing significant performance improvements in most metadata operations. Given the improvements and resiliency gained from enabling khepri, we've decided that, with the introduction of RabbitMQ 4.2, Khepri will now the default metadata store.
```

Comment:
We can replace "extensively tested" with "mature enough to become the default given its substantial data safety and recovery improvements over Mnesia".